### PR TITLE
Add CAVC lit support team

### LINKED ISSUE #15278: New Organization: CAVC Litigation Support Team
## User or job story
User story: As a Litigation Support team member who only works on CAVC cases, I need my own separate queue in Caseflow, so that I do not work or see any tasks that do not pertain to CAVC. 

## Acceptance criteria
- [ ] Include screenshot(s) in the Github issue if there are front-end changes, showing new org
- [ ] Update documentation: https://github.com/department-of-veterans-affairs/caseflow/wiki/Organizations
- [ ] New organization added to Caseflow for CAVC Litigation Support Team

### Background/context

Not all users in the Litigation Support team work all tasks. As the team that only works CAVC cases has never migrated to Caseflow we will want to include them as a new organization so they are not inadvertently assigned tasks they are not authorized to work or vice versa. 

### Other notes
image

### PR DESCRIPTION
**Pull request by yoomlam**
Resolves #15278 

### Description
Added a new organization called "CAVC Litigation Support" for the new team that handles CAVC-related tasks.

### Acceptance Criteria
- [ ] Include screenshot(s) in the Github issue if there are front-end changes, showing new org
- [ ] Update documentation: https://github.com/department-of-veterans-affairs/caseflow/wiki/Organizations
- [ ] New organization added to Caseflow for CAVC Litigation Support Team
- [ ] Reminder to self: https://github.com/department-of-veterans-affairs/caseflow/issues/15278#issuecomment-698472349

### Testing Plan
1. Create team in dev environment and check its information
```ruby
CavcLitigationSupport.singleton
```
2. Check the UI: http://localhost:3000/organizations/cavc-lit-support/users
3. Add and remove users in the UI and check in Rails console: 
```ruby
team=CavcLitigationSupport.singleton
team.users
````

Review new bullet for CavcLitigationSupport in https://github.com/department-of-veterans-affairs/caseflow/wiki/Organizations#existing-organizations

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---
(none) | image
(doesn't have CAVC Lit Support team) | image

### Code Documentation Updates
- [ ] Add or update code comments at the top of the class, module, and/or component.

### CODE REVIEW

File: app/models/organizations/cavc_litigation_support.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# For Litigation Support team members who only work on CAVC cases.
+
+class CavcLitigationSupport < Organization
+  def self.singleton
+    CavcLitigationSupport.first ||
+      CavcLitigationSupport.create(name: "CAVC Litigation Support", url: "cavc-lit-support")
+  end
+
+  def users_can_create_mail_task?
+    true
```
Comment (hschallhorn) [inline app/models/organizations/cavc_litigation_support.rb:14]: Not seeing where this is called out as an AC. Should the CavcLitigationSupport team be creating mail tasks?

Comment (yoomlam) [reply app/models/organizations/cavc_litigation_support.rb:14]: I'll remove it.

**Review by hschallhorn — APPROVED** [review]
Looks good!

**Comment by yoomlam** [comment]
Created new team in prod:
```ruby
Organization.count
=> 367
CavcLitigationSupport.singleton
=> #<CavcLitigationSupport:0x000000000aa22b00
 id: 477,
 type: "CavcLitigationSupport",
 name: "CAVC Litigation Support",
 role: nil,
 url: "cavc-lit-support",
 participant_id: nil,
 created_at: Mon, 19 Oct 2020 19:33:02 UTC +00:00,
 updated_at: Mon, 19 Oct 2020 19:33:02 UTC +00:00,
 status: "active",
 status_updated_at: nil,
 accepts_priority_pushed_cases: nil>
Organization.count
=> 368
```

Check at https://www.caseflowdemo.com/team_management (once demo is updated).

**Pull request merged**